core/src/main/java/org/veriblock/sdk/services/SerializeDeserializeService.java

```diff
@@ -362,7 +362,15 @@ public static VeriBlockBlock parseVeriBlockBlock(ByteBuffer buffer) {
     }
 
     public static VeriBlockBlock parseVeriBlockBlock(byte[] raw){
```

Comment:
Are you sure we need this parsing fix? It seems we expect length when parsing ByteBuffer and raw data when parsing byte[]. Maybe you should call the proper parsing method (or even hide parseVeriBlockBlock(byte[]) from public usage).

Reply:
parseVeriBlockBlock should be public, because it is used in tests